### PR TITLE
[fix] Add a checkbox for GSSoC contributors

### LINKED ISSUE #181: [OTHER] Add a GSSOC checkbox in issue template
### What would you like to share or ask?

Even though GSSOC labels are placed on raised issues, I think it would be also beneficial to add a GSSOC checkbox to the issue template(I talk about this more in [this issue](https://github.com/devs-in-tech/DevsInTech/issues/180)). I’ve implemented this approach as a maintainer for LinksHub, and it made GSSOC PRs and issues more distinguishable. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Closes #181 

## Changes proposed

I added a checkbox for GSSOC’23 contributors to confirm their participation in the program. 

## Screenshots

GSSOC checkbox in Docs issue template

## Note to reviewers
I would get errors when I try putting the checkbox in the Bug Report and Feature Request issue templates, so feedback on those two forms would be great @himanshu1221.

### CODE REVIEW

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @CBID2 I am unable to find `feature` issue template and another issue template apart from these follows and there is a label that is not working in the bug issue template 

Screenshot from 2023-06-01 19-10-07

Screenshot from 2023-06-01 19-12-31

**Comment by CBID2** [comment]
> Hey @CBID2 I am unable to find `feature` issue template and another issue template apart from these follows and there is a label that is not working in the bug issue template
> 
> Screenshot from 2023-06-01 19-10-07
> 
> Screenshot from 2023-06-01 19-12-31

Hi @himanshu1221! I made the change to the bug report and feature request template. I just have trouble adding the checkbox

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name: Feature Request 💡
-description: Have any new idea or new feature for LinksHub? Please suggest!
+name: "Feature Request 💡"
+description: "Have any new idea or new feature for Devsintech? Please request!"
```
Comment (Susmita-Dey) [inline .github/ISSUE_TEMPLATE/feature-request.yml:2]: Change the text "Devsintech" to "DevsInTech"

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Also, update your branch

**Comment by Susmita-Dey** [comment]
Hey @himanshu1221 kindly check this PR once.

**Review by prnvtripathi — APPROVED** [review]
LGTM :)

**Pull request merged**